controllers/gatling_controller.go

```diff
@@ -922,6 +938,34 @@ func (r *GatlingReconciler) getPodResources(gatling *gatlingv1alpha1.Gatling) co
 	return resources
 }
 
+func (r *GatlingReconciler) getObjectMeta(gatling *gatlingv1alpha1.Gatling) *metav1.ObjectMeta {
+	objectmeta := metav1.ObjectMeta{}
+	if &gatling != nil && &gatling.ObjectMeta != nil {
+		objectmeta = gatling.ObjectMeta
+	}
+	return &objectmeta
+}
+
+// Determine whether the label is attached to the runner or the reporter.
+func add_labels_pods(pod_type string, pod_obectmeta map[string]string) map[string]string {
+	if pod_obectmeta != nil {
+		if strings.Contains(pod_type, "runner") == true {
+			pod_obectmeta["type"] = "runner"
+		} else if strings.Contains(pod_type, "reporter") == true {
+			pod_obectmeta["type"] = "reporter"
+		}
+		return pod_obectmeta
+	} else {
+		pod_obectmeta := map[string]string{}
+		if strings.Contains(pod_type, "runner") == true {
+			pod_obectmeta["type"] = "runner"
+		} else if strings.Contains(pod_type, "reporter") == true {
+			pod_obectmeta["type"] = "reporter"
+		}
+		return pod_obectmeta
+	}
+}
```

Comment:
@S-mishina 
it seems to me that `add_labels_pod` function is for too specific purpose, by which I mean that it can be like for more generic or utility-like purpose as what it does is simply to append custom labels to map[string]string. 

Here is my suggestion
- create a generic helper function like `AddMapValue(value string, dataMap map[string]string) map[string]string` in [pkg/util/util.go](https://github.com/st-tech/gatling-operator/blob/main/pkg/utils/utils.go)
- use the AddMapValue to add `xxx-runner` or `xxx-reporter` label like this: 
 
```go
ObjectMeta: metav1.ObjectMeta{
	Name:        r.getObjectMeta(gatling).Name,
	Labels:      AddMapValue(gatling.Name+"-reporter", r.getObjectMeta(gatling).Labels),
	Annotations: r.getObjectMeta(gatling).Annotations,
},
```

Reply:
this is just an idea.  AddMapValue that i suggested previously can be like 

```go
// key  -  key to add to the map
// value  -  value to add to the map
// dataMap - the map to be updated
//  overwrite - flag to determine if value is overwritten when the key already exists in the map
AddMapValue(key string, value string,, dataMap map[string]string, overwrite bool) map[string]string
```

Reply:
@yokawasa 
Thanks for the review!
I thought this was a good suggestion
I've corrected it in this commit and I hope you will check it.
https://github.com/st-tech/gatling-operator/pull/63/commits/57e3a9dee1f89959449b4cff6c025f46bfd0956c

Reply:
@yokawasa 
Thanks for the review!
I thought this was a good suggestion
I did not convey the intent to myself, so I revised it again.
I hope you will check it.
https://github.com/st-tech/gatling-operator/pull/63/commits/c5152d97f439565057e6d8b0f1f577074474591a
https://github.com/st-tech/gatling-operator/pull/63/commits/c56755dcc9ac7eaa66f7e247945942a3cf562b2c

consultation
```
Labels:      utils.AddMapValue("type", gatling.Name+"-reporter", r.getObjectMeta(gatling).Labels),
```
I thought it would be better to know only the runner or reporter, not the pod name, so I did that.
What do you think?

Reply:
> Labels:      utils.AddMapValue("type", gatling.Name+"-reporter", r.getObjectMeta(gatling).Labels),
> I thought it would be better to know only the runner or reporter, not the pod name, so I did that.
What do you think?

@S-mishina 
Sorry, my mistake. You're right. It's supposed to be like this:
```
utils.AddMapValue("type", "runner", r.getObjectMeta(gatling).Labels),
or 
utils.AddMapValue("type", "reporter", r.getObjectMeta(gatling).Labels),
```

Reply:
@yokawasa @S-mishina 
I don't understand why the overwrite option is necessary.
May I ask for a detailed reason?

Reply:
@ksudate `AddMapValue` is util function and overwrite option give its user to determine if the value should be overwritten or not when the key for the value already exists in the map.  This is not necessary option for now but for future use